### DISCUSSION TITLE
Keeping past days in XML generated

### DISCUSSION THREAD

**Discussion by JoeyJoeyJoeJoeJr** [iptv-org/community#1034] [Q&A]
Quick question for folks - is there a way to keep past data in the xml file - so it appends instead of replaces?  The goal for me is to build an xml file that has historic and future data.

So hypothetically keep 10 historic days in the xml file and then add the new content being pulled daily.

Thanks!

**Comment by freearhey** [comment]
Probably the easiest way would be to change the name of the output file each day and then merge them together if needed.
